UndertaleModLib/Models/UndertaleRoom.cs

```diff
@@ -581,15 +572,15 @@ public class Background : UndertaleObject, INotifyPropertyChanged, IDisposable
         /// <remarks>
         /// This attribute is UMT-only and does not exist in GameMaker.
         /// </remarks>
```

Comment:
repating what i said in discord:
> its not fine cause "TileX" isnt public and thus not viewable.

Reply:
Something that'd better would be something along the lines of:
```
GameMaker uses an integer value for this, but since it treats everything that isn't `0` as `1`, it was decided to treat it here as a boolean for convenience.
```

Reply:
Is it fine?
```xml
/// <remarks>
/// Internally, GameMaker uses an integer value for storing the state: <c>0</c> - <see langword="false"/>, any other number - <see langword="true"/>.
/// This property is a wrapper that's made for convenience.
/// </remarks>
```

Reply:
```
for storing the state, where <c>0</c> acts as <see langword="false"/> and any other number as <see langword="true"/>. This property is a wrapper for it, which is made for convenience.
```